src/main/resources/META-INF/resources/primefaces/photocam/photocam.js

```diff
@@ -1137,11 +1158,31 @@ PrimeFaces.widget.PhotoCam = PrimeFaces.widget.BaseWidget.extend({
                     PrimeFaces.ajax.Request.handle(options);
                 }
             });
+            
+            $this = this;
+            Webcam.on("error", this.onCameraError);
 
             Webcam.attach(this.id);
             this.attached = true;
         }
     },
+    
+    /**
+     * Default error handler for webcam events
+     * @param {() => void} errorObj Error object containing message, stacktrace and so on.
```

Comment:
Just a quick note regarding the types: As far as I can tell from the name and how it's used, the type of `errorObj` should probably be `Error` instead of `() => void`. Cross-checking with the source code of PhotoCam, `WebcamError` and `FlashError` inherit from the builtin `Error` class.

Also, you can use `@private` to mark a method as private. Since this is an internally used callback, it qualifies in my opinion.